pstoremgr/pstoremgr_test.go

```diff
@@ -4,21 +4,21 @@ import (
 	"context"
 	"os"
 	"testing"
+	"time"
 
-	"github.com/ipfs/ipfs-cluster/api"
+	"github.com/ipfs/ipfs-cluster/test"
 
 	libp2p "github.com/libp2p/go-libp2p"
+	peer "github.com/libp2p/go-libp2p-peer"
 	ma "github.com/multiformats/go-multiaddr"
 )
 
-var pid = "QmXZrtE5jQwXNqCJMfHUTQkvhQ4ZAnqMnmzFMJfLewuabc"
```

Comment:
Cleaned up these tests a bit to not work with strings. Only added one tests at the end.